raiden/ui/console.py

```diff
@@ -332,11 +332,11 @@ def show_events_for(self, token_address, peer):
             events (list)
         """
         # Obtain the token manager
```

Comment:
Also change the comment to reflect that the token manager is now called the graph/channelgraph